### DISCUSSION TITLE
How do we expand the list of Nephio OWNERS?

### DISCUSSION THREAD

**Discussion by efiacor** [nephio-project/nephio#484] [Development]
High level discussion and ideas gathering related to how Nephio contributors can be granted elevated privileges on a given repo.

All/most existing Nephio repos have the following OWNERS file: 
https://github.com/nephio-project/nephio/blob/main/OWNERS

The OWNERS lists have not been updated in quite some time and each file should be reviewed at least every release.
See cleanup section here - https://www.kubernetes.dev/docs/guide/owners/#cleanup

If new approvers need to be assessed on their viability, how do we do that? GH stats, num of commits/PRs?

Some other relevant guidelines here - 
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Comment by henderiw** [comment]
So let's create a process how we proceed? Seems the most urgent is adding people to the OWNER files
-> define the criteria's to be added to the OWNERs file.
-> # PR review, code understanding (unit tests versus actual code contributions, etc)

**Reply by liamfallon** [reply]
I would argue that unit tests are code contributions, especially in a test-first world.

For example, I am raising a PR to fix the bug on the missing hidden file in Porch. The first thing I did was write a unit test to reproduce the fault.

**Comment by electrocucaracha** [comment]
Do we have defined criteria for considering someone an approver or reviewer? The CNCF Glossary has some documents that we can use, [Contributor ladder](https://glossary.cncf.io/contributor-ladder/) which includes also the emeritus process

**Reply by liamfallon** [reply]
@electrocucaracha this is a useful reference, I think we could use it.

**Comment by liamfallon** [comment]
Maybe we could start with opening up the docs repo to more owners? There is likely to be a good few PRs in the next days as we review the documents.

**Comment by liamfallon** [comment]
Addressed in recent months.
